lib/active_model/serializer.rb

```diff
@@ -111,6 +108,200 @@ def self.serialization_adapter_instance
       @serialization_adapter_instance ||= ActiveModelSerializers::Adapter::Attributes
     end
 
+    # Configuration options may also be set in
+    # Serializers and Adapters
+    config.collection_serializer = ActiveModel::Serializer::CollectionSerializer
+    config.serializer_lookup_enabled = true
+
+    # @deprecated Use {#config.collection_serializer=} instead of this. Is
+    #   compatibilty layer for ArraySerializer.
+    def config.array_serializer=(collection_serializer)
+      self.collection_serializer = collection_serializer
+    end
+
+    # @deprecated Use {#config.collection_serializer} instead of this. Is
+    #   compatibilty layer for ArraySerializer.
+    def config.array_serializer
+      collection_serializer
+    end
+
+    config.default_includes = '*'
+    config.adapter = :attributes
```

Comment:
See [Adapter unification RFC](https://github.com/rails-api/active_model_serializers/issues/1843) and the difference between 'attributes' and 'json', in terms of serializing individual records, should be whether `include_root_in_json` or `root` is true. In terms of generating the response document, when `root` is true, then other options like `meta` could be processed; Otherwise, they should be ignored since there's no notion of a document (distinct from the serialized resource(s)) without a root, and hence no document-level meta.